### PR TITLE
Fix for hardcoded link to localhost

### PR DESCRIPTION
**Pull request by ncksllvn**
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14959

### CODE REVIEW

**Comment by ncksllvn** [comment]
The Heroku instance may be deceiving, because we overwrite `www.va.gov` to use our current environment, such as `staging.va.gov`. In this case, Heroku is detected as a localhost, so the link looks unchanged, but it'll be processed correctly this time.

**Comment by bethpotts** [comment]
@ncksllvn Since there's no way to verify this link in the heroku instance, I'm not sure how to validate this?

**Comment by bethpotts** [comment]
@ncksllvn copied and pasted the link to va.gov and it looks good!

File: fragments/home/news.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - img: /img/homepage/secva.jpg
   alt: Video of VA Secretary Robert Wilkie's video announcing the new VA.gov.
-  title: VA.Gov Revised by Veterans
+  title: Welcome to the New VA.gov!
```
Comment (ncksllvn) [inline fragments/home/news.yml]: Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14963

File: pages/careers-employment/index.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ hublinks:
         to civilian jobs, and access other career resources.
       external: true
       target: _blank
-    - url: https://localhost/jobs/?utm_source=jobs_button&utm_campaign=ChooseVA_Website_Buttons
+    - url: "https://www.va.gov/jobs/?utm_source=jobs_button&utm_campaign=ChooseVA_Website_Buttons"
```
Comment (jbalboni) [inline pages/careers-employment/index.md]: We should remove all the query params here

**Pull request merged**